### PR TITLE
CHK-468: Add getLogo query

### PR DESCRIPTION
**Pull request by wdsrocha**
#### What problem is this solving?

Adds a query that allows the retrieval of the provider logo URL.

#### How should this be manually tested?

- [Workspace](https://w0geo--checkoutio.myvtex.com/cart/add?sku=289)
- Go to shipping
- Start to type a valid address. You can use "Praia de b".
- Assert that the `powered by Google` logo is displayed on bottom right of the suggestion list

#### Screenshots or example usage

You should see this:

### CODE REVIEW

File: graphql/schema.graphql [hunk 1]
```diff
@@ -2,4 +2,6 @@ type Query {
   suggestAddresses(searchTerm: String!): [AddressSuggestion!]!
 
   getAddressByExternalId(id: String!): Address!
+
+  getLogo: Image!
```
Comment (lucasecdb) [inline graphql/schema.graphql]: just a suggestion, can we rename this to `providerLogo` or something?

Comment (wdsrocha) [reply graphql/schema.graphql]: The query to `getProviderLogo` or the type to `ProviderLogo`? If you meant the query, I agree

Comment (lucasecdb) [reply graphql/schema.graphql]: the query to `providerLogo`, without the `get` prefix

Comment (wdsrocha) [reply graphql/schema.graphql]: Kay.

Any opinions on the other queries as well? Recently I noticed that it's not very common to name queries like functions, but at the same time, I found no nomenclature guidelines or something like that.

Comment (lucasecdb) [reply graphql/schema.graphql]: I'd rename the `getAddressByExternalId` as well, but that would be a breaking change right? I like to name the queries after the "object" I'm querying for (for example, if we had a query to get the current logged user I'd name it as something like `me`, `viewer` or `currentUser`), so for the `getByExternalId` I'd rename it as `address` and add the `externalId` as a parameter (like a filter).

I don't think there is any nomenclature guide 😂 but I follow something in the lines above

**Review by lucasecdb — APPROVED** [review]

**Review by alvim — APPROVED** [review]
I wasn't able to test in Safari... but I don't think it has something to do with this feature.

However, I believe we should focus some effort to fix all these "Error rendering extension point" in Safari for two reasons:

1. Because it has to work.
2. Because we could also be leaving some browser related errors behind in every feature we don't test and we will be charged for that when we release it.

**Pull request merged**